maven-archetype/src/main/resources/archetype-resources/readme/impl-guide.md

```diff
@@ -0,0 +1,60 @@
+#  Ozone ${distributionName} - Implementer Guide
+
+This distribution can be run using the Ozone Docker project, which is the default configuration for this. The quick start command below is for demonstration and trial purposes and would not be suitable for a stable environment.
+
+## Quick Start
+
+Build
+```bash
+./mvnw clean package
```

Comment:
`./mvnw clean package`
Not sure anymore, do we install Maven Wrapper in root or in `scripts/`

Reply:
Scripts now. Fixed.